src/transformers/models/cohere/modeling_cohere.py

```diff
@@ -1240,3 +1242,87 @@ def _reorder_cache(past_key_values, beam_idx):
                 tuple(past_state.index_select(0, beam_idx.to(past_state.device)) for past_state in layer_past),
             )
         return reordered_past
+
+
+@add_start_docstrings(
+    """
+    The Cohere Model transformer with a token classification head on top (a linear layer on top of the hidden-states
+    output) e.g. for Named-Entity-Recognition (NER) tasks.
+    """,
+    COHERE_START_DOCSTRING,
+)
+class CohereForTokenClassification(CoherePreTrainedModel):
```

Comment:
This looks like it was copied from another model (like Llama). Could we add a '# Copied from' statement like elsewhere in this file?

So something like
```
# Copied from transformers.models.llama.modeling_llama.LlamaForTokenClassification with Llama->Cohere
```